debezium-server/debezium-server-core/src/main/java/io/debezium/server/DebeziumServer.java

```diff
@@ -129,7 +129,7 @@ else if (beans.size() > 1) {
         configToProperties(config, props, PROP_KEY_FORMAT_PREFIX, "key.converter.", true);
         configToProperties(config, props, PROP_VALUE_FORMAT_PREFIX, "value.converter.", true);
         configToProperties(config, props, PROP_SINK_PREFIX + name + ".", DatabaseHistory.CONFIGURATION_FIELD_PREFIX_STRING + name + ".", false);
-        configToProperties(config, props, PROP_SINK_PREFIX, PROP_SINK_PREFIX, true);
+        configToProperties(config, props, PROP_SINK_PREFIX + name + ".", "offset.storage." + name + ".", false);
```

Comment:
Could you please introduce a constant for `offset.storage` prefix?

Reply:
Where should it be placed?